libgeos/libgeos_handle.go

```diff
@@ -0,0 +1,236 @@
+package libgeos
+
+/*
+#cgo linux CFLAGS: -I/usr/include
```

Comment:
This will be an issue for Mac Catalina because App has remove this folder entirely, so may be good to have an issue for that?

Reply:
Hmm, this seems like a major problem... I'm ok with this not working on Windows, but it seems like a major problem if it doesn't work on Mac. I'll find a solution for this before merging.

Reply:
I think I've found a solution. Thanks for pointing out this problem :)